lms/services/user.py

```diff
@@ -36,6 +36,8 @@ def store_lti_user(self, lti_user: LTIUser):
         else:
             self.db_session.add(new_user)
 
+        return existing_user or new_user
```

Comment:
I'm reusing "store_lti_user" but I might make more sense in a real version to use a `.get_current` or similar that it's lru_cache'ed.